src/EdpGithub/Http/Client.php

```diff
@@ -2,15 +2,16 @@
 
 namespace EdpGithub\Http;
 
+use Zend\EventManager\EventManager;
 use Zend\EventManager\EventManagerAwareInterface;
 use Zend\EventManager\EventManagerInterface;
-use Zend\Http\Client as HttpClient;
```

Comment:
@gianarb 

Not sure what your opinion is, but I personally try to avoid aliasing.